src/content/developers/tutorials/testing-ERC20-tokens-with-waffle/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Testing smart contracts with Waffle
+title: Testing ERC20 tokens with Waffle
```

Comment:
```suggestion
title: Testing ERC-20 tokens with Waffle
```
Let's try to stick with a single convention: `ERC-20`

Reply:
Along with conventions, please keep the filename lowercase (i.e. `src/content/developers/tutorials/testing-erc-20-tokens-with-waffle/index.md`).

Reply:
Also please add `"erc-20"` as a tag to this tutorial 😄